homeassistant/components/sensor/luftdaten.py

```diff
@@ -133,13 +133,9 @@ def device_state_attributes(self):
         except KeyError:
             return
 
-    @asyncio.coroutine
-    def async_update(self):
+    async def async_update(self):
         """Get the latest data from luftdaten.info and update the state."""
-        try:
-            yield from self.luftdaten.async_update()
-        except TypeError:
```

Comment:
Removed this type error because that would happen if Throttle was returning `None`…